include/snowplow/emitter/emitter.cpp

```diff
@@ -33,18 +33,18 @@ const int post_stm_bytes = 22;     // "stm":"1443452851000"
 
 #if defined(__APPLE__)
 #include "../http/http_client_apple.hpp"
-unique_ptr<HttpClient> createDefaultHttpClient() {
+unique_ptr<HttpClient> createDefaultHttpClient(const string &curl_cookie_file) {
   return unique_ptr<HttpClient>(new HttpClientApple());
 }
 #elif defined(WIN32) || defined(_WIN32) || defined(__WIN32) && !defined(__CYGWIN__)
```

Comment:
Perhaps, if we make use of these directives more often we could consider to create a macro that cover all the platform options. Something like:
`#define _SP_WIN_ (defined(WIN32) || defined(_WIN32) || defined(__WIN32) && !defined(__CYGWIN__))`
Just an idea - not sure this syntax is correct 🤔